labgrid/driver/usbvideodriver.py

```diff
@@ -46,7 +67,18 @@ def get_qualities(self):
                 ("mid", "video/x-h264,width=1280,height=720,framerate=25/1"),
                 ("high", "video/x-h264,width=1920,height=1080,framerate=25/1"),
                 ])
-        raise InvalidConfigError("Unknown USB video device {:04x}:{:04x}".format(*match))
+        else:
```

Comment:
My linter says: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it.